{{cookiecutter.project_slug}}/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,86 @@
+// For format details, see https://containers.dev/implementors/json_reference/
+{
+    "name": "{{cookiecutter.project_slug}}_dev",
+    "dockerComposeFile": [
+        "../local.yml"
+    ],
+    "init": true,
+    "mounts": [
+        {
+            "source": "./.history/bash_history",
+            "target": "/home/dev-user/.bash_history",
+            "type": "bind"
+        },
+        {
+            "source": "~/.ssh",
+            "target": "/tmp",
+            "type": "bind"
+        },
+        {
+            "source": "~/.ssh",
+            "target": "/home/dev-user/.ssh",
+            "type": "bind"
+        }
+    ],
+    // Tells devcontainer.json supporting services / tools whether they should run
+    // /bin/sh -c "while sleep 1000; do :; done" when starting the container instead of the container’s default command
+    "overrideCommand": true,
+    "service": "django",
+    // "remoteEnv": {"PATH": "/home/dev-user/.local/bin:${containerEnv:PATH}"},
+    "remoteUser": "dev-user",
+    "workspaceFolder": "/app",
+    // Set *default* container specific settings.json values on container create.
+    "customizations": {
+        {% if cookiecutter.editor == "VS Code" %}
+        "vscode": {
+            "settings": {
+                "editor.formatOnSave": true,
+                "[python]": {
+                    "analysis.autoImportCompletions": true,
+                    "analysis.typeCheckingMode": "basic",
+                    "defaultInterpreterPath": "/usr/local/bin/python",
+                    "editor.codeActionsOnSave": {
+                        "source.organizeImports": true
+                    },
+                    //"editor.defaultFormatter": "ms-python.black-formatter",
```

Comment:
Should we uncomment that?

Reply:
Looks like it's not working: https://github.com/microsoft/vscode-remote-release/issues/8474

Added a note about it